src/server/api/controllers/admin/RequestController.ts

```diff
@@ -0,0 +1,7 @@
+import { Get, JsonController, UseBefore, Res, Req, Post, Body } from 'routing-controllers';
+
+
+@JsonController('/request')
```

Comment:
What is this for again?

Reply:
I think i pushed this by mistake, its for admins to interact with requests